editor/src/core/workers/parser-printer/parser-printer.ts

```diff
@@ -527,8 +534,9 @@ function jsxElementToJSXExpression(
   element: JSXElementChild,
   imports: Imports,
   stripUIDs: boolean,
+  parentIsJSX: boolean,
```

Comment:
as far as I can see, this function is used where all parents are JSX elements, so unless there's a clear future need for the opposite, maybe this could be removed here? (keeping it in `jsxElementToExpression` instead, of course)

Reply:
my comment is related to `jsxElementToJSXExpression`, isn't this it?

Reply:
Gah, these functions are too closely named, this is `jsxElementToJSXExpression`.

Reply:
Fixed in bfa4f21ee.